tidb-cloud/changefeed-sink-to-mysql.md

```diff
@@ -72,6 +84,10 @@ The **Sink to MySQL** connector can only sink incremental data from your TiDB cl
     Finished dump at: 2020-11-10 10:40:20
     ``` 
 
+### Create a target table in MySQL
+
+You need to create a target table in MySQL with the same name as the source table in TiDB in advance. Otherwise, the data will not be replicated.
```

Comment:
```suggestion
You need to create a target table in MySQL with the same name as that of the source table in TiDB in advance. Otherwise, the data will not be replicated.
```